libs/ui/src/components/atoms/typography/typography.tsx

```diff
@@ -0,0 +1,111 @@
+import { font } from '../../../theme/global/font';
+import styled from 'styled-components';
+
+////////////////////PARAGRAPH TEXT////////////////////
+export const ParXs = styled.p`
+  font-size: ${font.size.xs};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+// ParSm Not actually implemented in design
+export const ParSm = styled.p`
+  font-size: ${font.size.sm};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+export const ParMd = styled.p`
+  font-size: ${font.size.md};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+export const ParLg = styled.p`
+  font-size: ${font.size.lg};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+export const ParXl = styled.p`
+  font-size: ${font.size.xl};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+////////////////////HEADER TEXT////////////////////
+export const H6 = styled.h6`
+  font-size: ${font.size.md};
+  font-weight: ${font.weight.black};
+  font-family: ${font.family.body};
+`;
+export const H5 = styled.h5`
+  font-size: ${font.size.lg};
+  font-weight: ${font.weight.bold};
+  font-family: ${font.family.body};
+`;
+export const H4 = styled.h4`
+  font-size: ${font.size.xl};
+  font-weight: ${font.weight.bold};
+  font-family: ${font.family.body};
+`;
+export const H3 = styled.h3`
+  font-size: ${font.size.xxl};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+export const H2 = styled.h2`
+  font-size: ${font.size.xxxl};
+  font-weight: ${font.weight.light};
+  font-family: ${font.family.body};
+`;
+export const H1 = styled.h1`
+  font-size: ${font.size.xxxxl};
+  font-weight: ${font.weight.light};
+  font-family: ${font.family.body};
+`;
+////////////////////DATA TEXT////////////////////
+export const DataXs = styled.p`
+  font-size: ${font.size.xs};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+export const DataSm = styled.p`
+  font-size: ${font.size.sm};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+export const DataMd = styled.p`
+  font-size: ${font.size.md};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+export const DataLg = styled.p`
+  font-size: ${font.size.lg};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+export const DataXl = styled.p`
+  font-size: ${font.size.xl};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
```

Comment:
Should we merge these into a single component every other style is the same? Or possibly even inherit from a base Data component

Reply:
I updated to so that they inherit. Definitely removed some code.

Reply:
I think we should try to inherit shared base styles. This feels like it's pretty large. What is the reason for wanting to avoid using props? I'm open to it, but curious since I've always preferred things like size to be handled at the prop level. If we go with the components approach, I think we should try to inherit these. 

Edit: I saw that you updated to use inheritance.

Reply:
Here's my reasoning for single component types. 

1) Simpler components. 
2) Easier to read
3) Faster to use, no typing in props. 
4) Limited functionality is a feature here. It enforces the styles out of the gate. 

All of these things fall in with line with the spec. I think we'd all had meetings about this too, right? The goal was to break down some of these primitives where we really don't need to be using props, but use single declarative props for things like button where props will always be a reality. 

Also worth noting that if we try this system and we really don't like it, we can compose all these components into larger Text components.

Reply:
Here's some additional reasoning that I think might help. 

```js 
const doAThing = (thing) => {
do(thing)
}

doAThing('thing1')
doAThing('thing2')
```

In the above example, we have a component that is maximally flexible and reusable. We could leave it as is. However we know that we're only going to do eight different things. The possibilities are finite,. 

In the spec, reliability, readability, and simplicity were key considerations for how this should be built. So does the function above best implement on our spec? 

```js
const thing1 = () => {
     doAThing('thing1')
}
const thing2 = () => {
     doAThing('thing2')
}
```
Instead of being maximally reusable or flexible, thing1() is better at being declarative, readable, and it's literally impossible to mess up. Also, the possibility narrowing has uses beyond fudging up the arguments , it constrains possibilities and helps ensure that we are in sync with the design system.  

Here's an additional consideration when it comes to DRY. Right now, on this PR, there's some extra code. However, we're not factoring all the needless props repeating that will happen in the future. I'd argue this is long-term more dry. 

A potential downside is that remembering to use <ParMd/> would be harder to remember to use <Text />. This is a true con. Also, it's a pattern that is less common. One component to handle them all would be easier to remember. But at the moment, I don't think the cons outweigh the pros.